Packs/AgariPhishingDefense/Integrations/AgariPhishingDefense/AgariPhishingDefense.yml

```diff
@@ -0,0 +1,456 @@
+        Example: 2020-05-01 \n2020-05-01T00:00:00\n2 days\n5 hours."
+      isArray: false
+      name: start_date
+      required: false
+      secret: false
+    - default: false
+      description: "The latest date time (UTC) a search should target (ISO 8601 format).\n\
+        Formats accepted: YYYY-MM-dd\nYYYY-MM-ddTHH:mm:ss\nN days\nN hours\nExample:\
+        \ 2020-05-01 \n2020-05-01T00:00:00\n2 days\n5 hours."
+      isArray: false
+      name: end_date
+      required: false
+      secret: false
+    - default: false
+      description: A comma-delimited list of optional fields to add to the default
+        payload. Additional fields would add data in the entry context.
+      isArray: true
+      name: add_fields
+      required: false
+      secret: false
+    - default: false
+      description: A comma-delimited list of fields to remove from the default payload.
+        Limited fields would return limited data in entry context.
+      isArray: true
+      name: rem_fields
+      required: false
+      secret: false
+    - default: false
+      description: A comma-delimited list of fields to include in the payload. Limited
+        fields would return limited data in entry context.
+      isArray: true
+      name: fields
+      required: false
+      secret: false
+    - default: false
+      defaultValue: '25'
+      description: The maximum number of items to be returned in the paged response.
+      isArray: false
+      name: limit
+      required: false
+      secret: false
+    - default: false
+      defaultValue: '1'
+      description: To page through a collection of message data.
+      isArray: false
+      name: page_id
+      required: false
+      secret: false
+    - default: false
+      description: A comma-delimited string that specifies the field ordering to be
+        applied to the response.
+      isArray: true
+      name: sort
+      required: false
+      secret: false
+    - default: false
+      description: |-
+        Search using advanced search syntax.
+        Format: field operator operand {and/or field operator operand}
+        Example: has_attachment=true and ip='10.0.0.0'
+        sbrs in [3.5, 2.6]
+        domain_reputation is not null
+        sbrs gt 3
+        sbrs>=3 and domain_tags eq internal
+      isArray: false
+      name: search
+      required: false
+      secret: false
+    deprecated: false
+    description: Retrieves a list of messages.
+    execution: false
+    name: apd-list-message-data
+    outputs:
+    - contextPath: AgariPhishingDefense.Message.has_attachment
+      description: Has attachment.
+      type: Boolean
+    - contextPath: AgariPhishingDefense.Message.ip
+      description: IP address.
+      type: String
+    - contextPath: AgariPhishingDefense.Message.message_id
+      description: The Global message ID.
+      type: String
+    - contextPath: AgariPhishingDefense.Message.ptr_name
+      description: PTR name.
```

Comment:
I suggest considering a more elaborate description for some of the context paths, for example (but not limited to) this one, as iit could really boost the usability of your integration.

Reply:
@guykeller sure will make this change